docs/components/query.md

```diff
@@ -255,7 +255,7 @@ Query node exposing PromQL enabled Query API with data retrieved from multiple
 store nodes.
 
 Flags:
-      --alert.query-url=ALERT.QUERY-URL
+      --alert.query-url=ALERT.QUERY-URL  
```

Comment:
Docs check fails due to these whitespaces I think. You can fix these by running `make changed-docs`!

Reply:
Thanks! Fixed the docs.